client/src/components/CallPaneReceiver.js

```diff
@@ -0,0 +1,172 @@
+// import React, { useRef, useEffect } from 'react';
```

Comment:
`CallPaneReceiver` and `CallPaneCaller` are really similar, you can make it more general so it can be reused in several places. This way, when you fix/change something you'll have less problems caused by forgetting to replicate something.

Reply:
Agreed, will refactor